tests/integrationtest/r/executor/issues.result

```diff
@@ -868,3 +868,137 @@ set @@tidb_max_chunk_size = default;
+      └─TableFullScan_17	256.00	0	cop[tikv]	table:t	NULL	keep order:true, stats:partial[c:unInitialized]	<memory>	<disk>
+select @@tidb_partition_prune_mode;
+@@tidb_partition_prune_mode
+dynamic
+explain analyze select * from pt order by id;  # expected distsql concurrency 15
+id	estRows	actRows	task	access object	execution info	operator info	memory	disk
+TableReader_11	256.00	256	root	partition:all	max_distsql_concurrency: 15	NULL	<memory>	<disk>
+└─TableFullScan_10	256.00	256	cop[tikv]	table:pt	NULL	keep order:true	<memory>	<disk>
+explain analyze select * from pt limit 100;    # expected distsql concurrency 7
+id	estRows	actRows	task	access object	execution info	operator info	memory	disk
+Limit_7	100.00	100	root	NULL	NULL	offset:0, count:100	<memory>	<disk>
+└─TableReader_11	100.00	107	root	partition:all	max_distsql_concurrency: 7	NULL	<memory>	<disk>
+  └─Limit_10	100.00	199	cop[tikv]	NULL	NULL	offset:0, count:100	<memory>	<disk>
+    └─TableFullScan_9	100.00	199	cop[tikv]	table:pt	NULL	keep order:false	<memory>	<disk>
+explain analyze select * from pt limit 100000; # expected distsql concurrency 15
+id	estRows	actRows	task	access object	execution info	operator info	memory	disk
+Limit_7	256.00	256	root	NULL	NULL	offset:0, count:100000	<memory>	<disk>
+└─TableReader_11	256.00	256	root	partition:all	max_distsql_concurrency: 15	NULL	<memory>	<disk>
+  └─Limit_10	256.00	256	cop[tikv]	NULL	NULL	offset:0, count:100000	<memory>	<disk>
+    └─TableFullScan_9	256.00	256	cop[tikv]	table:pt	NULL	keep order:false	<memory>	<disk>
+explain analyze select * from pt where val = 125 limit 100;     # expected distsql concurrency 15
+id	estRows	actRows	task	access object	execution info	operator info	memory	disk
+Limit_8	100.00	0	root	NULL	NULL	offset:0, count:100	<memory>	<disk>
+└─TableReader_13	100.00	0	root	partition:all	max_distsql_concurrency: 15	NULL	<memory>	<disk>
+  └─Limit_12	100.00	0	cop[tikv]	NULL	NULL	offset:0, count:100	<memory>	<disk>
+    └─Selection_11	100.00	0	cop[tikv]	NULL	NULL	eq(executor__issues.pt.val, 125)	<memory>	<disk>
+      └─TableFullScan_10	125.00	0	cop[tikv]	table:pt	NULL	keep order:false, stats:partial[val:missing]	<memory>	<disk>
+explain analyze select * from pt where val = 125 limit 100000;  # expected distsql concurrency 15
+id	estRows	actRows	task	access object	execution info	operator info	memory	disk
+Limit_8	204.80	0	root	NULL	NULL	offset:0, count:100000	<memory>	<disk>
+└─TableReader_13	204.80	0	root	partition:all	max_distsql_concurrency: 15	NULL	<memory>	<disk>
+  └─Limit_12	204.80	0	cop[tikv]	NULL	NULL	offset:0, count:100000	<memory>	<disk>
+    └─Selection_11	204.80	0	cop[tikv]	NULL	NULL	eq(executor__issues.pt.val, 125)	<memory>	<disk>
+      └─TableFullScan_10	256.00	0	cop[tikv]	table:pt	NULL	keep order:false, stats:partial[val:missing]	<memory>	<disk>
+explain analyze select * from pt order by id limit 100;         # expected distsql concurrency 7, but currently get 1, see issue #55190
```

Comment:
Known issue https://github.com/pingcap/tidb/issues/55190
Will not be fixed in this PR.